src/internal/_reduced.js

```diff
@@ -1,3 +1,7 @@
 module.exports = function(x) {
-  return x && x.__transducers_reduced__ ? x : {value: x, __transducers_reduced__: true};
+  return x && x['@@transducer/reduced'] ? x :
```

Comment:
This one doesn't really count, as the comment relates to code not modified in this pull request.